css/properties/min-width.json

```
@@ -252,9 +252,15 @@
                   "version_added": "2"
                 }
               ],
-              "safari_ios": {
-                "version_added": null
-              },
+              "safari_ios": [
+                {
+                  "version_added": "11"
+                },
+                {
+                  "alternative_name": "min-intrinsic",
```

This is just intrinsic on safari. Why would it be min-intrinsic on iOS?

```suggestion
                  "alternative_name": "intrinsic",
```